contracts/cheddar_5x5_tic_tac_toe/src/game.rs

```diff
@@ -82,13 +86,11 @@ impl Game {
         assert_eq!(self.players[1], player_2.clone());
 
         assert_eq!(
-            self.players[0].piece,
-            self.board.current_piece,
+            self.players[0].piece, self.board.current_piece,
```

Comment:
if we remove `Player` structure, then we can have a helper function: `fn player_piece(index: u8)`